src/app/zap-templates/templates/app/CHIPClientCallbacks-src.zapt

```diff
@@ -428,6 +452,28 @@ bool IMDefaultResponseCallback(const chip::app::Command * commandObj, EmberAfSta
     return true;
 }
 
+bool IMWriteResponseCallback(const chip::app::WriteClient * writeClient, EmberAfStatus status)
```

Comment:
Why do we have less info than for `IMReadReportAttributesResponseCallback` where the signature is ` IMReadReportAttributesResponseCallback(const app::ReadClient * apReadClient, const app::ClusterInfo & aPath, TLV::TLVReader * apData, Protocols::InteractionModel::ProtocolCode status)`

I would expect it to carry `aPath` too ? Or otherwise I would expect `IMReadReportAttributesResponseCallback` to not carry `aPath` but uses `readClient->GetSourceNodeId();`  and so the `GET_ATTRIBUTE_RESPONSE_CALLBACKS` may be used for both if something smart if found for the `tvlFilter` case.

Reply:
The write response is a status report with path and it does not contain cluster specific data.
You can implement `InteractionModelDelegate::WriteResponseStatus`, which comes with the Path and WriteClient for GetSourceNodeId().

Reply:
Good idea for `IMReadReportAttributesResponseCallback`, the `IMReadReportAttributesResponseCallback` should be able to not use the info from path, however, I'm not sure if this will break some test cases which relies on the stdout from `IMReadReportAttributesResponseCallback` since it will print the info from path.